source/iface/device_image.rst

```diff
@@ -7,3 +7,45 @@
 **********************
 ``sycl::device_image``
 **********************
+
+::
+
+  template <bundle_state State>
+  class device_image;
+
+.. seealso:: |SYCL_SPEC_DEVICE_IMAGE|
```

Comment:
This doesn't currently link to the subsection.  Only to the beginning of the spec.